const unexpectedError = {
  isSuccess: false,
  userMessage: 'Unexpected Error, Please Try Again'
}

export default unexpectedError;